.github/workflows/ScriptTests.yml

```diff
@@ -22,28 +22,30 @@ jobs:
         steps:
             - name: "Checking changed files"
               id: files
-              uses: jitterbit/get-changed-files@v1
+              uses: softprops/diffset@v1
+              with:
+                global_files: |
```

Comment:
You're getting a warning about this global_files. I think here you want to use special_files, right?

Reply:
The documentation says you can use everything _files, the statement works, but the gh action gives a warning..

Reply:
Extracted from the docs:

This action supports dynamically named inputs which will result in dynamically named outputs. Specifically this action accepts any inputs with a suffix of _files

Reply:
Yeah, I also saw that issue about this: https://github.com/softprops/diffset/issues/5
There's some "limitation" in the actions related to that.